package.json

```diff
@@ -115,7 +115,7 @@
     "react-router-redux": "^4.0.8",
     "react-transition-group": "^4.4.5",
     "react-use": "^17.5.1",
-    "react-virtualized": "^9.22.3",
+    "react-virtualized": "9.22.5",
```

Comment:
Lock this to 9.22.5 as 9.22.6 completely fails CI. It's likely not following semver.